config/crd/bases/logging.openshift.io_clusterloggings.yaml

```diff
@@ -368,6 +368,25 @@ spec:
                             description: TimeUnit is a time unit like h,m,d
                             pattern: ^([0-9]+)([yMwdhHms]{0,1})$
                             type: string
+                          namespaceSpec:
+                            description: The per namespace specification to delete documents older than a given minimum age
+                            items:
+                              properties:
+                                minAge:
+                                  description: Delete the records matching the namespaces which are older than this MinAge (e.g. 1d)
```

Comment:
is there a `maxAge` somewhere in ES CRD? if there is no min/max then just name it something like `agelimit`, or `maxAge`
If this is used to delete older namespaces, then calling it a minimum of some quantity sounds confusing.